packages/cactus-common/src/main/typescript/signer-key-pairs.ts

```diff
@@ -0,0 +1,26 @@
+import crypto from "crypto";
+import secp256k1 from "secp256k1";
+
+export interface ISignerKeyPairs {
+  privateKey: any;
+  publicKey: any;
```

Comment:
Would type this according to what it actually is (I think it was a buffer or Uint8Array, but can't remember for sure), just avoid using the any type.

Reply:
Update data type.